src/bin/server.rs

```diff
@@ -21,6 +21,19 @@ fn main() -> Result {
         return static_web_server::settings::cli_output::display_version();
     }
 
+    if let Ok(path) = std::env::var("SWS_GENERATE") {
+        let path = std::path::PathBuf::from(path);
+        let mut completions_path = path.clone();
+        completions_path.push("completions");
+        clap_allgen::render_shell_completions::<static_web_server::settings::cli::General>(
+            completions_path,
+        )?;
+        let mut man_path = path.clone();
+        man_path.push("man");
+        clap_allgen::render_manpages::<static_web_server::settings::cli::General>(man_path)?;
+        return Ok(());
```

Comment:
We should log every operation with some `tracing::info` to hint users.

Reply:
sounds good to me, will do.